user-interface/src/data-verification/ConsolidationOrderModal.tsx

```diff
@@ -104,20 +121,36 @@ function ConsolidationOrderModalComponent(
   function hide() {
     if (modalRef.current?.hide) {
       modalRef.current?.hide({});
+      setConsolidationType(null);
+      administrativeConsolidationRef.current?.checked(false);
+      substantiveConsolidationRef.current?.checked(false);
+      setLeadCaseDivisionCode('');
+      leadCaseDivisionRef.current?.clearValue();
+      setLeadCaseNumber('');
+      leadCaseNumberRef.current?.clearValue();
     }
   }
 
+  function handleSelectConsolidationType(ev: React.ChangeEvent<HTMLInputElement>) {
+    setConsolidationType(ev.target.value as ConsolidationType);
+  }
+
   function handleLeadCaseInputChange(ev: React.ChangeEvent<HTMLInputElement>) {
     const { caseNumber, joinedInput } = validateCaseNumberInput(ev);
-    leadCaseIdRef.current?.setValue(joinedInput);
+    leadCaseNumberRef.current?.setValue(joinedInput);
     if (caseNumber) {
       setLeadCaseNumber(caseNumber);
-      modalRef.current?.buttons?.current?.disableSubmitButton(false);
     } else {
-      modalRef.current?.buttons?.current?.disableSubmitButton(true);
+      setLeadCaseNumber('');
     }
   }
 
+  useEffect(() => {
+    modalRef.current?.buttons?.current?.disableSubmitButton(
+      consolidationType === null || leadCaseNumber.length !== 8,
```

Comment:
We don't need to necessarily, but if we're doing something like this length check in other places we might define a well-named constant to indicate why we're checking for a length of 8.